pxtsim/runtime.ts

```diff
@@ -563,13 +571,43 @@ namespace pxsim {
 
     export type EventIDType = number | string;
 
+    class EventHandler {
+        private busy = 0;
+        constructor(public handler: RefAction, public flags: number) {}
```

Comment:
This is a bit pedantic, but wondering if the `flags` parameter will be able to pass on multiple flags or just one at a time since it has type number?

Reply:
i'm matching the codal naming here, but actually numbers can hold up to 32 flags (one for each bit)